src/test/java/com/iexec/core/worker/WorkerServiceTests.java

```diff
@@ -80,32 +118,32 @@ void shouldReturnZeroForAllCountersWhereNothingHasAppended() {
         Gauge aliveTotalCpuGauge = Metrics.globalRegistry.find(WorkerService.METRIC_CPU_TOTAL_GAUGE).gauge();
         Gauge aliveAvailableCpuGauge = Metrics.globalRegistry.find(WorkerService.METRIC_CPU_AVAILABLE_GAUGE).gauge();
 
-        Assertions.assertThat(aliveWorkersGauge).isNotNull();
-        Assertions.assertThat(aliveTotalCpuGauge).isNotNull();
-        Assertions.assertThat(aliveAvailableCpuGauge).isNotNull();
-
-        Assertions.assertThat(aliveWorkersGauge.value()).isZero();
-        Assertions.assertThat(aliveTotalCpuGauge.value()).isZero();
-        Assertions.assertThat(aliveAvailableCpuGauge.value()).isZero();
+        assertThat(aliveWorkersGauge).isNotNull();
```

Comment:
with the test modification, you didn't take the opportunity to use assertAll ?

Reply:
Please see aeb00e1